### ISSUE TITLE
Box Menu 4 error with framework 3

### ISSUE THREAD

**Issue by tomgreenfield** [adaptlearning/adapt_framework#2343]
Box Menu 4 breaks the loading of framework 3 with the following error:
```console
adaptView.js:9 Uncaught TypeError: Cannot read property 'get' of undefined
    at n.attributes (adaptView.js:9)
    at Function.h.result (underscore.min.js:5)
    at n.resultExtend (underscore.results.js:18)
    at n.resultExtendInstance [as resultExtend] (backbone.controller.results.js:27)
    at n.attributes (adapt-contrib-boxmenu.js:13)
    at Function.h.result (underscore.min.js:5)
    at n._ensureElement (backbone.min.js:2)
    at n.e.View (backbone.min.js:2)
    at n [as constructor] (backbone.min.js:2)
    at n [as constructor] (backbone.min.js:2)
```

This is due to value inheritance only being [fixed](https://github.com/adaptlearning/adapt_framework/pull/2123/commits/ef0f5de6c8180e9562d5c15479226ae6c610c1e1) for framework 4.

Suggest we release a new version of Box Menu which doesn't use `resultExtend`, for (non-accessible) compatibility with framework 3. We can then pop it back in at some point in the future with a bumped framework dependency.

**Also note use of `a11y_aria_image`: https://github.com/adaptlearning/adapt-contrib-boxmenu/commit/3b9f7dc03395512d0febd56949da3dcabb7647e6**.

**Comment by oliverfoster** [comment]
Sounds sensible enough..... ...

**Comment by brian-learningpool** [comment]
Corrected in yesterday's re-tag of Box Menu v4.0.0.
